### ISSUE TITLE
Note content not visible on Android

### ISSUE THREAD

**Issue by Botono** [standardnotes/forum#888]
I use the Plus Editor for all my notes. I can view and edit all notes in the web, Linux, and Mac apps, but not on Android. All my notes are listed, but they all appear to have no content unless I switch to the Plain Editor and then I can see the content with HTML.

**Comment by mobitar** [comment]
Are you using a custom theme? Could you try disabling any themes and trying again?

**Comment by bryvin** [comment]
Additionally, if @mobitar's suggestion doesn't work; try long-pressing on the theme and choosing "Redownload".

**Comment by qwerquestion** [comment]
I have a similar issue here:

It started a couple of weeks ago that Plus Editor notes on my android app appear blank (see attached screenshot). Notes that I have created a while ago are still displayed on my windows Standard Notes app. However, notes that I have created on my Android using Plus Editor after February 5. 2020 are not displayed anywhere!

These notes also remain blank when switching to another editor.

I am worried about my notes and conncerned with continuing using the app.

I hope someone can help?

![Screenshot_20200226_115751_com standardnotes](https://user-images.githubusercontent.com/61501513/75342269-5e616000-5896-11ea-96c6-6f7c586b6f49.jpg)
